src/main/scala/se/nimsa/sbx/seriestype/SeriesTypeDAO.scala

```diff
@@ -187,4 +187,14 @@ class SeriesTypeDAO(val dbConf: DatabaseConfig[JdbcProfile])(implicit ec: Execut
     } yield st
     innerJoin.result
   }
+
+  def seriesTypesForListOfSeries(seriesIds: Seq[Long]): Future[SeriesIdSeriesTypesResult] = db.run {
```

Comment:
Sorry for nitpicking. Can this method return `Future[Seq[SeriesIdSeriesType]]` instead and leave the wrapping in a `SeriesIdSeriesTypesResult` to `SeriesTypeServiceActor`?